### ISSUE TITLE
Process not ended when closing the application with the window control X in KDE.

### ISSUE THREAD

**Issue by cdysthe** [theinvisible/openfortigui#1]
When closing the application by clicking the window control X in KDE an 'openfortigui' process is still running and has to be killed before the application can be opened again. If I close it using Menu > File > Exit the application closes as it should. It would be good to be able to close the application with the windows controls. 

Running KDE Neon with KDE 5.9.5.

**Comment by boospy** [comment]
In KDE 5.8.5 LTS it is working how it should. When i click the X the Window closes to the infopanel. When you clicking the X in Window the App schould to be closed, only minimized to tray.

**Comment by cdysthe** [comment]
Maybe I was not being clear. The application does close, but there's a process called 'opnfortigui" still running without the UI. The application UI is closed. When I try to open it again I get an error saying "This application is already running. Exiting now". I then kill the 'openfortigui' process and I can open the application again. This ONLY happens if I close the application with the window control, not when I use the Menu dropdown and Exit. I have tried many times and see the same result.

**Comment by boospy** [comment]
Ok, strange. I'am not able to close the application with the windowcontrol. I only can close the application with the applicationmenu. With windowcontroll i only can close it to the tray. What system to you use. I have here Kubuntu 16.04 with the KDE PPA and the OpenFortiguipackage from "styrion.at/apt" Normaly it should be in the tray, but you must edit systemtray --> applications --> Entries and edit visibly from the app from automaticly to visible/always.

**Comment by cdysthe** [comment]
Thanks! I had to open the application, then open the systray settings. There was no entry for openfortiGUI. I then selected "show all" and I did show up. Then I set it to "shown" instead of "auto" so now it works as it should. All other applications are either "hidden" or "auto", but I have to set openfortiGUI to "shown" to be able to see it in the systray for some reason. I run KDE Neon which is based on Ubuntu LTS and currently has KDE 5.9.5.
